packages/gestalt/src/ButtonToggle.tsx

```diff
@@ -204,7 +237,6 @@ const ButtonToggleWithForwardRef = forwardRef<HTMLButtonElement, Props>(function
     [styles.thumbnailMd]: size === 'md' && graphicSrc,
     [styles.thumbnailSm]: size === 'sm' && graphicSrc,
     [styles[color]]: !disabled && !selected,
-    [touchableStyles.tapCompress]: !disabled && isTapping,
```

Comment:
why did we removed this one? Should have stayed only in `!isInVRExperiment`

Reply:
It didn't have any effect. Tap compression is controlled by transform CSS property from `useTapFeedback`.

Reply:
Does this mean we can clean all the isTapping from other components?

Reply:
i think so, yes